### PR TITLE
Add view for Africa firms

### PR DESCRIPTION
**Pull request by kajah**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ Of note: Oftentimes the discussion of and efforts to address harassment and disc
 
 <iframe class="airtable-embed" src="https://airtable.com/embed/shrUkljNsrKY515xw?backgroundColor=orange&viewControls=on" frameborder="0" onmousewheel="" width="100%" height="533" style="background: transparent; border: 1px solid #ccc;"></iframe>
 
+## [](#vcs-mf) VCs #MovingForward - Africa
```
Comment (triketora) [inline index.md]: the anchor tag should be different than the one below

**Comment by triketora** [comment]
best practice for frontend changes is to also include a screenshot so reviewers can visually verify

File: index.md (hunk 1 above)
Comment (triketora) [reply index.md]: and probably you want this one to have something like "africa" in it

**Comment by kajah** [comment]
Screen Shot 2020-08-23 at 11 27 25 PM

**Pull request merged**